### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by samwebster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,20 @@
...
-TIMES is an energy systems model generator from the International Energy Agency that is used around the world to inform energy policy - https://iea-etsap.org/index.php/etsap-tools/model-generators/times
+The project began as a [Hackathon](https://news.microsoft.com/life/hackathon/) project hence the repo being in the Microsoft organisation, however Microsoft are not funding this work. Further work has been done as part of company Hackathons and the rest in people's own time (both Microsoft employees and not).
+
+TIMES is an energy systems model generator from the International Energy Agency that is used around the world to inform energy policy - https://iea-etsap.org/index.php/etsap-tools/model-generators/times. Detailed documentation is available at https://iea-etsap.org/index.php/documentation
+
+## Overview
+
+The input data is in multiple Excel files and spread across multiple worksheets. Each worksheet can contain any number of data 'tables'. These are annotated by a cell directly above with a tilde and the name of the table e.g. ~FI_T (flexible import table).
+
+Steps:
```
Comment (siddharth-krishna) [inline README.md:15]:
```suggestion
Steps performed by this tool:
```

File: README.md [hunk 2]
```diff
@@ -4,7 +4,20 @@
...
+- Load the input Excel files
+- Locate and extracts all of the 'tables' into Python objects which include the data in a Pandas DataFrame
```
Comment (siddharth-krishna) [inline README.md:17]:
```suggestion
- Locate and extract all of the 'tables' into Python objects which include the data as Pandas DataFrames
```

**Review by siddharth-krishna — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -4,7 +4,20 @@
 
 This project is an open source tool to convert TIMES model Excel input files to DD format ready for processing by GAMS.  The intention is to make it easier for people to reproduce research results on TIMES models.
 
-TIMES is an energy systems model generator from the International Energy Agency that is used around the world to inform energy policy - https://iea-etsap.org/index.php/etsap-tools/model-generators/times
+The project began as a [Hackathon](https://news.microsoft.com/life/hackathon/) project hence the repo being in the Microsoft organisation, however Microsoft are not funding this work. Further work has been done as part of company Hackathons and the rest in people's own time (both Microsoft employees and not).
```
Comment (tminka) [inline README.md:7]: I don't think you should include this sentence.  What message are you trying to convey?

Comment (samwebster) [reply README.md:7]: That the project isn't funded by Microsoft. People could be forgiven for thinking the opposite given it's location

Comment (tminka) [reply README.md:7]: Why does anyone care about that?  Is it something to do with the level of support?

Comment (samwebster) [reply README.md:7]: That's the most important reason but also I've put a lot of my own time into this that I'm not happy for Microsoft to get the credit for

Comment (tminka) [reply README.md:7]: The "own time" sentence covers that.

Comment (samwebster) [reply README.md]: Is this better? Update doesn't show for me at the top of this conversation for some reason

**Review by tminka — APPROVED** [review]

**Pull request closed without merging**